lib/vanagon/platform/windows.rb

```diff
@@ -54,8 +54,50 @@ def generate_packaging_artifacts(workdir, name, binding)
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_msi_packaging_artifacts(workdir, name, binding)
         FileUtils.mkdir_p(File.join(workdir, "wix"))
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.wxs.erb"), File.join(workdir, "wix",  "#{name}.wxs"), false, { :binding => binding })
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.filter.xslt.erb"), File.join(workdir, "wix", "#{name}.filter.xslt"), false, { :binding => binding })
+        currentwd = Dir.getwd
+
+        # Recurse through the list of files in the resources/windows/wix directory and copy them over to the work directory.
+        # Treat special cases as necessary (e.g. erb files)
+        # Could probably do this as a function except for xslt/project special cases
+        Dir.chdir("resources/windows/wix")
+        Dir.glob("**/*.*").each do |source_file|
+          src_dirname = File.dirname(source_file)
+          src_basename = File.basename(source_file)
+          target_dir = File.join(workdir, "wix", src_dirname)
+          FileUtils.mkdir_p(target_dir)
```

Comment:
The number of files is not expected to be large, but it allows us to differentiate between special case erb files.

Would be quite happy to only glob for erb files if we could copy the rest except for the erb using a cp-r operation.